web/gds-user-ui/src/modules/dashboard/certificate/lib/trisaImplementationValidationSchema.ts

```diff
@@ -15,15 +15,12 @@ const commonNameRegex =
 
 export const trisaImplementationValidationSchema = yup.object().shape({
   testnet: yup.object().shape({
-    endpoint: yup
-      .string()
-      .notRequired()
-      .matches(trisaEndpointPattern, {
-        message: _i18n._(t`TRISA endpoint is not valid.`)
-      }),
+    endpoint: yup.string().matches(trisaEndpointPattern, {
+      message: _i18n._(t`TRISA endpoint is not valid.`)
+    }),
     common_name: yup
       .string()
-      .notRequired()
+      .required()
```

Comment:
This should allow the Trisa implementation to have an error state if the user do not entered required field